src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+and `scroll`. Except for the scroll event, this is largely dependent on the device used for
+scrolling: touch events are dispatched when scrolling with the finger on mobile devices, while wheel
+events occur when scrolling with a mouse wheel. The scroll events are fired after initial scrolling
+has completed. And in general, no DOM event blocks scrolling, unless the website uses [non-passive
+event listeners](/uses-passive-event-listeners). So we think of scrolling as decoupled from DOM
+Events altogether. What we want to measure is the time from when the user moves enough to produce a
+scroll gesture until the first frame that shows that scrolling happened.
+
+#### How to define the latency of an interaction?
+
+As we noted above, interactions that have a "down" and "up" component need to be considered
+separately in order to avoid attributing the time the user spent holding their finger down.
+
+For these types of interactions, we’d like the latency to involve the durations of all events
+associated with them. Since event durations for each "down" and "up" part of the interaction can
+overlap, the simplest definition of interaction latency that achieves this is the maximum duration
+of any event associated with it. Referring back to the keyboard diagram from earlier, this would be
+the `keydown` duration, as it is longer than the `keyup`:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/WFPm4W86CqhFsNc1UZuW.jpeg", alt="Keyboard interaction
+with maximum duration highlighted", width="800", height="311" %}
+
+The `keydown` and `keyup` durations may overlap as well. This may happen for instance when the frame
+presented for both events is the same, as in the following diagram:
+
+{% Img src="image/kns0INkO77RkiEStzHWYrugyWj32/e7htYAZb44AW4UeplBwJ.jpeg", alt="Keyboard interaction
+where press and release occur in the same frame", width="800", height="311" %}
+
+There's are pros and cons to this approach of using the maximum, and we're interested in [hearing
+your feedback](#feedback):
+
+* **Con**: It does not capture the full wait time of the user. For instance, it will capture the
+  start or end of a drag, but not both.
+* **Pro**: It is aligned with how we intend to measure scroll in that it only measures a single
+  duration value.
+* **Pro**: It aims to reduce noise for cases like keyboard interactions, where the `keyup` usually
+  does nothing and where the user may execute the key press and release quickly or slowly.
+
+For scrolling (which just has a single associated event) we’d like to define its latency as the time
+it takes for the browser to produce the first frame as a result of scrolling. That is, the latency
+is the delta between the event `timeStamp` of the first DOM event (like `touchmove`, if using a
+finger) that is large enough to trigger a scroll and the first paint which reflects the scrolling
+taking place.
+
+### Aggregate all interactions per page
```

Comment:
To be honest... this section reads like it was an afterthought of the blogpost, which I guess it is.

The text below which explores some options and asks for feedback, I'm not sure are fleshed out enough to be actionable.  Not sure, but, I wonder if we can just say something like "Next steps: Aggregating these interactions into a single metric value, stay tuned!"

Reply:
It's not an afterthought. I already mention that `We’re exploring and evaluating several options.` to make it clear that this is ongoing work. I do see how it's not fleshed out enough to be actionable, but it's important to note this crucial step in the metric definition so I'm keeping this section.